builder/amazon/common/run_config.go

```diff
@@ -426,6 +440,9 @@ type RunConfig struct {
 	// 10m
 	WindowsPasswordTimeout time.Duration `mapstructure:"windows_password_timeout" required:"false"`
 
+	// Metadata Settings
```

Comment:
Once you updated the changes, you can make this line like:
```suggestion
	// [Metadata Settings](#metadata-settings) 
```
This will create a link to the Metadata Settings section.